core/src/main/java/org/apache/carbondata/core/datastore/filesystem/HDFSLocalCarbonFile.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.core.datastore.filesystem;
+
+import org.apache.hadoop.fs.FileStatus;
+
+public class HDFSLocalCarbonFile extends HDFSCarbonFile {
```

Comment:
Can use HDFSCarbonFile directly, no need of this calls as this class don;t have any of its own functionality.

Reply:
just to differentiate between actual HDFS file system and local file loading in the cluster where storage is hdfs, I have given this name which will help to get to know that loading in local. That is why I kept it like that

Reply:
But I feel no need to keep dummy wrappers, based on `HDFS_LOCAL` file type we will know it is HDFS local. 
I suggest to remove it.

Reply:
Ok, removed